libretro/libretro.cpp

```diff
@@ -1841,13 +1848,17 @@ float System_GetPropertyFloat(SystemProperty prop)
 
 bool System_GetPropertyBool(SystemProperty prop)
 {
+#if PPSSPP_PLATFORM(IOS)
+   return false;
+#else
```

Comment:
Would prefer this is inside the switch, in case we add more sysprops later.  It would be easy to not notice this ifdef and break iOS by accident.

-[Unknown]

Reply:
Good point, I agree. I've changed it. Thanks!